inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -0,0 +1,31 @@
+/*-- scss:rules --*/
+$web-font-path: "https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&family=Source+Code+Pro:ital,wght@0,400;0,600;1,400;1,600&display=swap" !default;
```

Comment:
@gregswinehart Do you think we could cut back these font weights a little bit?

Reply:
I suppose we could if it's a thing. That said, I would probably only lose the 800 weight? I like to celebrate the whole catalog....

Reply:
Thanks Greg. I'm going to leave this as is for now and will review again with Carson this week.